### ISSUE TITLE
Redirect Request: Removing Temporary Redirect for Form 26-4555 SAH

### ISSUE THREAD

**Issue by jennymayoco** [department-of-veterans-affairs/va.gov-team#79053]
### Status 
**[2024-07-15]** Code is ready to merge and deploy on Friday for a Monday morning (7/22) release.
**[2024-07-08]** Unblocked; target release date is 7/22/2024.
**[2024-05-17]** Jenny Mayo moved the ticket to blocked while they hold on a new relaunch date. She will be OOO 5/20-5/31 and asked Tricia Baker to please share any updates or questions here and tag Kristin. 
**[2024-04-16]** From Tricia Baker: the date to release this form has changed and has been pushed out and will not be released on 4/30.  I don't have an exact date but will keep you posted.
**[2024-03-26]** Add to Next Refinement so this will be poised to pull into a sprint when they give us the go-ahead.  Note that Jenny and tbaker are aware we'd like 2 weeks notice so we can prioritize.
  - From Jenny: The team is getting ready to relaunch this form and is projecting relaunching in mid-April. Testing is currently on hold, so this date might change. I wanted to create the ticket to remove the temporary redirect with enough time so we can align across teams and sprints for when the redirect is removed and when the online application is relaunched. 

### Instructions
- Requests must be submitted **AT LEAST** 2 weeks in advance.  Some requests take a significant amount of time and coordination to implement, so start the process as soon as you know you will need one. 
- This issue will be used from initial request through implementation to ensure all individuals working on this are notified of status updates. Please do not create multiple issues to track different steps.
- It is your responsibility to notify VA stakeholders as appropriate.  

### Type of request
- [ ] We are retiring or taking down a page and need to redirect the URL (complete redirect section)
- [ ] We are changing the URL of an existing page (complete redirect section)
- [ ] We need a custom vanity URL (complete vanity URL section)
- [X] Removing temporary redirect - There is currently a temporary 302 redirect for online application 26-4555, the [ticket is #16493](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16493) for the implementation work. The form will be relaunching soon with a projected target of mid-April, but with the possibilities of it changing.

### Implementation date
When does this request need to be live?
Choose one:
- [X] On a specific date, connected to a program or publishing requirement. **DATE TBD** - but we want to align the temp redirect removal with the relaunch of the form URL
- [ ] Any day prior to a deadline/latest possible date. **DEADLINE:** 
- [ ] Any day after an earliest possible date. **EARLIEST DATE:**
- [ ] During a specific date range: ***EARLIEST DATE TO LATEST DATE:***
- [ ] No hard timing requirements, publish when possible.

### Redirects 
Current URL  |  Redirect Destination or New URL
---  |  ---
 |  |
_Note: Removed URLs since we aren't implementing a redirect, but instead are removing a current temporary redirect._

### Redirect Removal
The temp redirect to remove is to: https://www.va.gov/housing-assistance/disability-housing-grants/how-to-apply/ (it's on  on the form URL:
https://www.va.gov/housing-assistance/disability-housing-grants/apply-for-grant-form-26-4555/introduction  and all child pages

### Vanity URLs 
Vanity URL  |  Landing page
---  |  ---
requested URL | landing page URL

**Link to campaign landing page request issue:**

### Process, Roles and Responsibilities
- [ ] Requesting team: Above information is provided
- [ ] Requesting team: All appropriate VA stakeholders are notified as appropriate
- [x] IA: Request is vetted and documented and implementation plan is clear 
- [x] IA: Request is assigned to appropriate team for implementation 
- [ ] Implementation team: Work is complete
- [ ] Implementation team: Validated in production
- [ ] Requesting team: Validates everything is correct in production and closes ticket

**Comment by jennymayoco** [comment]
Hi @jilladams @FranECross! The team is getting ready to relaunch this form and is projecting relaunching in mid-April. Testing is currently on hold, so this date might change. I wanted to create the ticket to remove the temporary redirect with enough time so we can align across teams and sprints for when the redirect is removed and when the online application is relaunched. Looping in @tbaker1026 who can share latest updates and coordinate with you all. Let me know if there are any questions!

cc @megzehn for visibiltiy

**Comment by FranECross** [comment]
@jennymayoco Thanks for the heads up and ticket! We look forward to coordinating with ya'll. cc @jilladams

**Comment by FranECross** [comment]
@jennymayoco Just nudging with a request (if possible) that we have at least two weeks notice of the release date so that we can refine and prioritize the ticket. Thanks in advance! cc @jilladams

**Comment by jennymayoco** [comment]
@FranECross thank you for flagging! ^^^ @tbaker1026 for awareness that timeline is at least 2 weeks notice of release date

**Comment by jennymayoco** [comment]
@tbaker1026 bringing your comment from the [original ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16493) here: "@jilladams at the latest we will go live with this form on 4/30 as long as testing will go well.  I will update this ticket to confirm the date." cc @FranECross

**Comment by FranECross** [comment]
Thanks for the heads-up, @jennymayoco and @tbaker1026 ! We'll include this in our next refinement (next week) and have it poised to pull into a sprint when we get the final word from you. cc @randimays @chriskim2311 @jilladams

**Comment by jilladams** [comment]
@tbaker1026 latest on timing for removing the temporary redirect is in this ticket ^. Our next sprint planning is Wed 4/17 to plan work through 4/30, so sounds like Fran is prepping us to pull it into that sprint.

**Comment by jennymayoco** [comment]
@tbaker1026 what does that lift look like to relaunch the form? Is it an instant change?

**Comment by FranECross** [comment]
@jennymayoco We were looking at the description on this ticket, and it looks like dupe of the temp redirect that we need to remove. Will you please review and correct if needed. Thanks in advance.

**Comment by jennymayoco** [comment]
Hi @FranECross, just to confirm, are you referring to the current URL and the destination URL in the ticket?

**Comment by jennymayoco** [comment]
got it thanks for confirming @FranECross! I copied the URLs from [the original temp redirect ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16493) since we are requesting to remove the temporary redirect on these URLs. How should I present removing a temporary redirect?

**Comment by FranECross** [comment]
@jennymayoco I added an additional row under Type of Request and checked it. I think for this situation, perhaps not putting anything in the Current/Redirect fields, and instead put info next to the new line that I added stating which temporary redirect is being removed? @randimays @chriskim2311

**Comment by jennymayoco** [comment]
@FranECross that makes sense! I've updated the ticket. let me know if there's any other questions on the work for this ticket!

**Comment by tbaker1026** [comment]
@FranECross @jennymayoco - the date to release this form has changed and has been pushed out and will not be released on 4/30.  I don't have an exact date but will keep you posted.

**Comment by jennymayoco** [comment]
Moving this to blocked while we hold on a new relaunch date. I will be OOO 5/20-5/31, @tbaker1026 please share any updates or questions here and tag Kristin. Thanks!

**Comment by jennymayoco** [comment]
hi @jilladams @FranECross - moving this to in progress as we have a new relaunch date for the form, which will be 7/16 ([linking to Slack update](https://dsva.slack.com/archives/C044AGZFG2W/p1716317601227219?thread_ts=1716317495.628979&cid=C044AGZFG2W)).

@tbaker1026 - around what time would this change go into effect? We'd want to make sure that we can align when the PW team deploys redirects so there's minimal gap between the form going live again and removing the redirect.

cc @megzehn

**Comment by tbaker1026** [comment]
@jennymayoco Confirming with the housing team on timing

**Comment by jennymayoco** [comment]
Hi @FranECross @jilladams - the team is double checking with the housing team that the final launch date is 7/16. Since the forms team still has tasks to go through before they can turn the flipper on, they are looking at turning this feature on on 7/22 and so we were wondering if the temp redirect could be removed on 7/22, would that work?

@tbaker1026 @jeana-adhoc @Thrillberg if there are any other questions on aligning the flipper with the temp redirect removal, please share them here so that PW team has eyes on the latest updates. Bringing some initial questions from Tricia from Slack:

1. Should the team add the flipper to the supporting forms page to test it?
2. If the temp redirect is removed on 7/22, when would users be able to access the page without having the direct URL? (I think 10am et for that day's deploy but I wasn't sure if this would be a different process since this is a temp redirect removal)

I can also coordinate a quick chat with the forms and PW teams before launch so we're all aligned.

**Comment by tbaker1026** [comment]
@FranECross @jilladams I did confirm they are ready to go live on 7/16 so that is confirmed.

**Comment by FranECross** [comment]
@tbaker1026 Thanks for the update! I see in the description that the previous redirect implementation ticket is referenced, but the temporary redirect itself isn't explicitly called out. Will you please confirm that this is the redirect we are removing? https://www.va.gov/housing-assistance/disability-housing-grants/how-to-apply/

Also, if they are going live on 7/16, do you want the temp redirect removed at the earliest date of 7/17 / after?

Thanks in advance! cc @jilladams @randimays @chriskim2311

**Comment by tbaker1026** [comment]
@jennymayoco can you confirm the above redirect they are removing?  Yes the earliest on 7/22, thank you

**Comment by jennymayoco** [comment]
@FranECross the temp redirect is on the form URL: 
https://www.va.gov/housing-assistance/disability-housing-grants/apply-for-grant-form-26-4555/introduction  **and all child pages**

It's currently redirecting to: https://www.va.gov/housing-assistance/disability-housing-grants/how-to-apply/

**Comment by jennymayoco** [comment]
hi @randimays @jilladams ! I just wanted to check in and see if we're all set for 7/22? thank you for any updates!

cc @tbaker1026

**Comment by FranECross** [comment]
@jennymayoco I believe it's poised for release on 7/22. I see that it's in the pipeline Code Review/QA. @randimays  is OOO for a bit today, but can confirm when back in the office.

**Comment by tbaker1026** [comment]
@FranECross yes we are good for release on the 22nd.  They are releasing the new API tomorrow.

**Comment by randimays** [comment]
@jennymayoco @FranECross yes, we'll be ready to merge and deploy the code on Friday for a Monday morning release.

**Comment by jennymayoco** [comment]
awesome, thank you for confirming! 🙂

**Comment by randimays** [comment]
Redirect has been merged; it will deploy Monday morning at 9am CST.

**Comment by jennymayoco** [comment]
Thanks, @randimays ! I see the form URL doesn't redirect to the `/how-to-apply` anymore 👍 

@tbaker1026 @megzehn confirming that the form URL goes to the form intro page (404 at the moment)

**Comment by randimays** [comment]
Thanks @jennymayoco, please close this ticket when ready.

**Comment by jennymayoco** [comment]
Closing ticket as we've confirmed temp redirect has been removed. Thanks everyone!
